test/screenshot/spec/mdc-textfield/mixins/disabled-3.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
Can this fit inside `disabled-2.html`? Would like to reduce the number of screenshots if possible

Reply:
I think it might be able to, but I know we'll have to have another one for all the disabled color customization tests with prefix/suffix, so I thought it might be good to go ahead and start a new one.

Want me to try to fit into 2 still?

Reply:
Let's try to squeeze it in for now, prefix and suffix can be in their own thing when the time comes

Reply:
Hm they kinda fit, but they cause scrollbars to appear. Is this acceptable?

https://storage.googleapis.com/mdc-web-screenshot-tests/lizmitchell/2019/12/18/21_04_39_522/spec/mdc-textfield/mixins/disabled-2.html.windows_chrome_77.png

Reply:
Yeah, should be fine